pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -45,11 +45,18 @@ def parse_toml_option(
         if value is True:
             return [action.option_strings[0]]
         return []
+
     if isinstance(action, argparse._StoreAction):
         if isinstance(value, int):
             value = str(value)
         return [action.option_strings[0], value]
-    return []  # pragma: no cover
+
+    if isinstance(action, argparse._ExtendAction):  # type: ignore[attr-defined]
+        if isinstance(value, str):
```

Comment:
Shouldn't this be `if not`? Why are we casting a `str` to a `str`?

Reply:
actually ... it might not be needed at all, will fix that